e2e/_suites/fleet/fleet.go

```diff
@@ -1127,6 +1127,81 @@ func (fts *FleetTestSuite) upgradeAgent(version string) error {
 	return nil
 }
 
+func (tfs *FleetTestSuite) checkDataStream() error {
```

Comment:
This method is great! I can imagine up-coming tests for integrations in which this method would be abstracted so that it supports passing `exists` field names and `term` terms.

Reply:
yes!